matlab/Makefile

```diff
@@ -115,9 +115,15 @@ $(icetoolbox_file):: $(icethunk_target) $(slice2matlab_path) $(lang_srcdir)/lib/
 	cp -rf $(cpp_bindir)/slice2matlab $(lang_srcdir)/toolbox/build/
 	# Doc files
 	cp -rf $(lang_srcdir)/toolbox/doc $(lang_srcdir)/toolbox/build
+ifneq (,$(MATLAB_COMMAND))
```

Comment:
MATLAB_COMMAND is set by CI to the executable that we need to run. We were already using this in the tests, but there were a few additional places where we still used matlab directly.